static/src/javascripts/projects/common/modules/experiments/tests/remote-epic-variants.js

```diff
@@ -11,6 +12,9 @@ const remoteVariant: Variant = {
     canRun: () => true,
 };
 
+const geolocation = geolocationGetSync();
```

Comment:
I might be mis-remembering, but was there a thing before where we moved this as close as possible to the place where the geolocation is used, to give maximum time for it to get set?

Reply:
Ah I found the PR I was thinking of: https://github.com/guardian/frontend/pull/22322 I guess this is still relevant here.

Reply:
hey @tomrf1 good shout, I had completely forgotten about that! I guess it would make little difference here as the geolocation can be inferred from the edition (as AU has its own edition) but delaying the geo checks to the `canRun` function shouldn't have any downsides so worth doing.

Reply:
I should have mentioned @tjmw !